documentation/doxygen/src/ref_retarget_fs.txt

```diff
@@ -1,5 +1,7 @@
 /**
 \defgroup fs_interface_api File Interface
+
```

Comment:
It is also ok to have no empty line between \defgroup and \brief or \details. This doesn't impact the HTML output, but might be a bit easier to read the doxygen text file. We typically don't have such gaps.

Reply:
Ok, I've removed them.